suites/squid/rgw/tier-1_rgw_sw_qat.yaml

```diff
@@ -52,6 +52,10 @@ tests:
               args:
                 placement:
                   label: rgw
+                  nodes:
```

Comment:
Hi @SaiSuriya30 , can we also update the suite files which use conf : conf/squid/rgw/tier-0_rgw.yaml with node, if bootstrap testcase has node mentioned for rgw deployment